packages/table/components/Table.tsx

```diff
@@ -45,6 +51,12 @@ export class Table<T> extends React.PureComponent<TableProps, TableState> {
     }
   );
 
+  public _cache = new CellMeasurerCache({
+    defaultHeight: ROW_HEIGHT,
+    defaultWidth: 150,
+    fixedHeight: true
+  });
+
   public getColumnSizes = memoizeOne(
```

Comment:
It looks like we cache it twice with `memoizeOne` and with `this._cache`. Do they serve different purposes?

Reply:
What specifically do we cache twice?

Reply:
So the function itself is memoized with `memoizeOne` and it uses `this._cache` internally so it looks like we're memoizing the cache, am I missing something?

Reply:
Ahh I understand. `this._cache` (changed to `this.cellMeasureCache`) is what `react-virtualized` uses to calculate the width dynamically, but we need to get the value it returns in `getColumnSizes`